carRental/src/main/resources/db/changelog/0.0.0.1/202306222016_added_entity_Driver_License.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <property name="autoIncrement" value="true"/>
+    <property name="now" value="now()" dbms="postgresql"/>
+
+    <changeSet id="2023062019-1" author="ThaysVieira">
+        <createTable tableName="driver_license">
+            <column name="id" type="bigint" autoIncrement="${autoIncrement}">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="number" type="varchar">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issued_in" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issued_at" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="expiry_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
```

Comment:
addForeignKeyConstraint should be added on the sprat file

Reply:
@Esmaeeilenani what is this xml?

Reply:
this is for liquibase migration, we try to sprat table creation from FK constraints

Reply:
ah ok i never used liquibase